l4ping/l4ping.TESTGEN.ps1

```diff
@@ -0,0 +1,143 @@
+#===============================================
+# Script Input Parameters Enforcement
+#===============================================
+Param(
+    [parameter(Mandatory=$false)] [string] $Config = "Default",
+    [parameter(Mandatory=$true)]  [string] $DestIp,
+    [parameter(Mandatory=$true)]  [string] $SrcIp,
+    [parameter(Mandatory=$true)]  [ValidateScript({Test-Path $_ -PathType Container})] [String] $OutDir = "" 
+)
+$scriptName = $MyInvocation.MyCommand.Name 
+
+function input_display {
+    $g_path = Get-Location
+
+    Write-Output "============================================"
+    Write-Output "$g_path\$scriptName"
+    Write-Output " Inputs:"
+    Write-Output "  -Config     = $Config"
+    Write-Output "  -DestIp     = $DestIp"
+    Write-Output "  -SrcIp      = $SrcIp"
+    Write-Output "  -OutDir     = $OutDir"
+    Write-Output "============================================"
+} # input_display()
+
+#===============================================
+# Internal Functions
+#===============================================
+function banner {
+    [CmdletBinding()]
```

Comment:
Remove

Reply:
Leaving as is to match legacy behavior for all other TestGens